gkr_iop/src/precompiles/lookup_keccakf.rs

```diff
@@ -23,10 +32,12 @@ use crate::{
         GKRCircuit, GKRProof, GKRProverOutput,
         layer::{Layer, LayerType},
     },
-    precompiles::utils::{MaskRepresentation, not8_expr},
+    precompiles::utils::{
+        MaskRepresentation, not8_expr, set_slice_felts_from_u64 as push_instance,
```

Comment:
This `set_slice_felts_from_u64` serve as a better replacement of push_instance, for its take iterator instead of vector thus less memory allocation